Makefile

```diff
@@ -1,4 +1,7 @@
 test:
 	$(MAKE) -C tests test
 
-.PHONY: test
+test-complex:
```

Comment:
Rename `complex-tests` for consistency.

Reply:
I thought about that, but decided for test-complex, because I wanted:
* the target to start with test- for tab completion purposes etc.
* the name be somewhat consistent with what we just introduced for heketi, which is "test-functional" there.

Reply:
This project shouldn't care at all what heketi is doing. ;) That said, do you at least intend to rename all instances of `simple-tests` to `test-simple` as well? I'd like to see such changes as part of the same PR (in different commits, of course) but I'd be willing to take a promise of a subsequent PR instead.

...also when would we ever be able to use tab-completion for this?

Reply:
Of course the project should also care what heketi is doing, since it was partly spun off of that project, and heketi is one of the core components that this is a test- and integration framework for... ;-)

And are you seriously telling me, you have never used make's tab-completion? :-P

FWIW: changed the target to complex-tests.